libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/ConfigService.java

```diff
@@ -114,11 +112,14 @@ private String formatPath(String path, String... pathProperties) {
         return String.format(path, (Object[]) pathProperties);
     }
 
-    // PYIC-7048 Replace this with proper config
     public long getBearerAccessTokenTtl() {
-        return Optional.ofNullable(getEnvironmentVariable(BEARER_TOKEN_TTL))
-                .map(Long::valueOf)
-                .orElse(DEFAULT_BEARER_TOKEN_TTL_IN_SECS);
+        try {
+            return Long.parseLong(getParameter(ConfigurationVariable.BEARER_TOKEN_TTL));
+        } catch (ConfigParameterNotFoundException | NumberFormatException e) {
+            LOGGER.warn(
+                    LogHelper.buildErrorMessage("Failed to get bearer token ttl from config", e));
+            return DEFAULT_BEARER_TOKEN_TTL_IN_SECS;
```

Comment:
Do we actually need this default any more? Can we just add this as the configured value in all environments

Reply:
Yeh I guess I was being a bit cautious but have added them here https://github.com/govuk-one-login/ipv-core-common-infra/pull/1026 , will remove